src/ConservationChecker.jl

```diff
@@ -147,15 +147,15 @@ function check_conservation!(
 end
 
 """
-        check_conservation!(
-        cc::WaterConservationCheck,
-        coupler_sim,
-        get_slab_energy,
-        get_land_energy,
-        )
+    check_conservation!(
+    cc::WaterConservationCheck,
+    coupler_sim,
+    get_slab_energy,
+    get_land_energy,
+    )
 
-computes the total water ∫ ρq_tot dV of the various components
-of the coupled simulations, and updates cc with the values.
+computes the total water, ``\\int \\rho q_{tot} dV``, of the various components
```

Comment:
I actually prefer the existing format, at least for small doc strings like this-- it reads a bit simpler

Reply:
I prefer the math typesetting :)  What does @LenkaNovak prefer?

Reply:
I agree with Charlie. The original is more readable from the REPL.

Reply:
Oh, ok. I tend to look at the web version and I think the math typesetting makes it easier to read. It is ok, I can revert this change in a future PR

Reply:
Done in #223